enqueue/enqueue-bundle/0.3/post_install.txt

```diff
@@ -0,0 +1 @@
+[Enqueue] Run <comment>make consume</> command to consume messages
```

Comment:
I would remove this as the post install script if for things you need to do to make things work. It should not replace proper documentation of the package.